src/app/landscape/page.tsx

```diff
@@ -0,0 +1,65 @@
+import { data } from '@/app/data'
+import EcosystemMap from '@/components/ReactTreeMap'
+import { Metadata } from 'next'
+import type { Viewport } from 'next'
+
+export const viewport: Viewport = {
+  width: 'device-width',
+  initialScale: 1,
+}
+
+export const metadata: Metadata = {
+  metadataBase: new URL('https://chained-assets.vercel.app/landscape'),
```

Comment:
Extract out this common content in this and next file in a common utility file/function?

Reply:
sure. it should be useful for code reusing